universal-application-tool-0.0.1/app/controllers/PostgresController.java

```diff
@@ -46,4 +46,17 @@ public PostgresController(
             },
             httpExecutionContext.current());
   }
+
+  public Result syncAdd(String name) {
```

Comment:
I know I recommended the name `syncAdd`, but now that I see a better convention for differentiating sync/async versions of the same interface is to put sync on the end e.g. `addSync`. This is what Node.js and I think other tools with the same problem do.

Reply:
And now that I think about it a little more, we should standardize our HTTP controller method names that do standard actions. Standard methods are:
`index` - list the resource instances
`show` - display a specific resource instance
`new` - display a form for creating a new resource
`create` - create a new resource instance, usually redirect to the `show` method
`update` - update an existing resource instance
`destroy` - delete a resource instance

Reply:
Renamed functions to the standard. I leave the redirection logic out for now as for now we are simply using the controller as an example for how to interact with the database. We could add this when we have a create form.

Reply:
Sounds good!